indico/modules/events/templates/reviewing_questions_management.html

```diff
@@ -0,0 +1,105 @@
+{% from 'message_box.html' import message_box %}
+{% from '_sortable_list.html' import sortable_list %}
+
+<div class="manage-reviewing-questions">
+    <div class="toolbar space-after hide-if-locked">
+        <a class="i-button arrow icon-plus highlight" data-toggle="dropdown">
+            {%- trans %}Add new question{% endtrans -%}
+        </a>
+        <ul class="dropdown">
+            {% for name, field_type in field_types.iteritems() -%}
+                <li>
+                    <a class="js-action-button"
+                       data-href="{{ url_for(actions.create, event, field_type=name, **(args.create|default({}))) }}"
```

Comment:
if you get rid of the nested dict as suggested above, you can just use `**common_url_args` instead of the `default({})` stuff